uaclient/entitlements/landscape.py

```diff
@@ -74,6 +77,23 @@ def application_status(
                 messages.LANDSCAPE_SERVICE_NOT_ACTIVE,
             )
 
+    def applicability_status(
+        self,
+    ) -> Tuple[ApplicabilityStatus, Optional[messages.NamedMessage]]:
+        applicability_status = super().applicability_status()
+        if applicability_status[0] == ApplicabilityStatus.INAPPLICABLE:
+            affordance = self.entitlement_cfg["entitlement"].get(
+                "affordances", {}
+            )
+            affordance_series = affordance.get("series", None)
+            current_release = system.get_release_info()
+            if current_release not in affordance_series:
+                return (
+                    ApplicabilityStatus.INAPPLICABLE,
+                    messages.LANDSCAPE_INAPPLICABLE,
+                )
```

Comment:
Let's make it exactly the same as the super() impl of this check just to be safe
```
        if (
            self.affordance_check_series
            and affordance_series is not None
            and system.get_release_info().series not in affordance_series
        ):
            return (
                ApplicabilityStatus.INAPPLICABLE,
                messages.LANDSCAPE_INAPPLICABLE
            )

```